src/cmd/gz.cc

```diff
@@ -345,6 +353,80 @@ extern "C" void cmdTopicEcho(const char *_topic,
   }
 }
 
+//////////////////////////////////////////////////
+extern "C" void cmdTopicFrequency(const char *_topic)
+{
+  if (!_topic || std::string(_topic).empty())
+  {
+    std::cerr << "Invalid topic. Topic must not be empty.\n";
+    return;
+  }
+  using namespace std::chrono;
+  int count = 0;
+  int samples = 11;
+  int window = samples - 1;
+  std::vector<int64_t> timeV;
+  std::vector<float> intervalV;
+  float sum = 0.0;
+  float dev = 0.0;
+  float mean = 0.0;
+  float stdDev = 0.0;
+
+  std::function<void(const ProtoMsg&)> cb = [&](
+        __attribute__((unused)) const ProtoMsg &_msg)
```

Comment:
`__attribute__` is gcc/clang thing. It won't work on windows. The simplest thing is to remove the variable name
```suggestion
  std::function<void(const ProtoMsg&)> cb = [&](const ProtoMsg &)
```